index.bs

```diff
@@ -289,12 +426,18 @@ A {{FileSystemFileHandle}}'s associated [=FileSystemHandle/entry=] must be a [=f
 The <dfn method for=FileSystemFileHandle>getFile()</dfn> method steps are:
 
 1. Let |result| be [=a new promise=].
+1. Let |entry| be the result of [=locating an entry=]
+   given [=this=]'s [=FileSystemHandle/locator=].
```

Comment:
Can this happen synchronously on the main thread? This seems like something that should happen in parallel and in the future in the file system parallel queue. (This applies multiple times I think.)

(Not entirely sure how this works with async iteration. That already seems to assume it's running in parallel with how it runs "query access"...)

Reply:
Good point. I've moved this into the "in parallel" section, which we can then deal with in #95